README.md

```diff
@@ -1,21 +1,6 @@
-## Prototyping/Investigation Notes
+##AWS Batch WPS Implementation
```

Comment:
After merging this I notice that you need a s-pace in between after ## - there are a few more headings needing this below